media-video/ffmpeg/ffmpeg7-7.1.1.recipe

```diff
@@ -122,13 +120,13 @@ CONFLICTS_tools="
 
 PROVIDES_devel="
 	ffmpeg7${secondaryArchSuffix}_devel = $portVersionCompat
-#	devel:libavcodec$secondaryArchSuffix = $libavcodecVersionCompat
-#	devel:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
-#	devel:libavfilter$secondaryArchSuffix = $libavfilterVersionCompat
-#	devel:libavformat$secondaryArchSuffix = $libavformatVersionCompat
-#	devel:libavutil$secondaryArchSuffix = $libavutilVersionCompat
-#	devel:libswresample$secondaryArchSuffix = $libswresampleVersionCompat
-#	devel:libswscale$secondaryArchSuffix = $libswscaleVersionCompat
+	devel:libavcodec$secondaryArchSuffix = $libavcodecVersionCompat
+	devel:libavdevice$secondaryArchSuffix = $libavdeviceVersionCompat
+	devel:libavfilter$secondaryArchSuffix = $libavfilterVersionCompat
+	devel:libavformat$secondaryArchSuffix = $libavformatVersionCompat
+	devel:libavutil$secondaryArchSuffix = $libavutilVersionCompat
+	devel:libswresample$secondaryArchSuffix = $libswresampleVersionCompat
+	devel:libswscale$secondaryArchSuffix = $libswscaleVersionCompat
```

Comment:
I'm not sure if these should be enabled yet. Haiku itself still uses ffmpeg6, so it makes sense to keep that as the default version.